### DISCUSSION TITLE
Sorting SheetsSheet by when the sheet was created

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#2439] [Q&A]
I figured out that one of the things that were confusing me about the sheets' sheet is how the order of the sheets changes after I visit them. I would like the order of the sheets to always remain the same. Another way to phrase it is that I want the sheets ordered by when they were created, not by when they were last visited. I don't see any ordering at all in the columns of the sheets' sheet. Is it possible to achieve what I want?

**Comment by saulpw** [comment]
I think you want `g Shift+S` (`sheets-all`).  The shortcut column is the Alt+number to go directly to that sheet, and shows the order the sheets were created in. Use `Shift+Q` (`quit-sheet-free`) to quit a sheet and remove it from this All Sheets Sheet as well.

**Reply by cool-RR** [reply]
Ah, I liked the part where the sheets are ordered by when they are created, though I don't like that I have to quit sheets in a special way for them to be removed from this sheet. (I also assume that `quit-sheet-free` prevents `undo-last-quit`. ) Is there any way to get one without the other?

**Reply by saulpw** [reply]
You can rebind `q` to `quit-sheet-free` if you want.  But you're right that it would prevent `undo-last-quit`.

The Sheets Sheet has given people trouble all along, so I could see it being replaced with something more like what you're asking for.  The best thing to do is find or create a wishlist item explaining everything you want here, so that it's on our radar for the flurry of activity before a release.

**Reply by cool-RR** [reply]
I think I talked about this before, but I can't find it. What I'd really like is a tree display that shows which sheet was spawned from which other sheet. But of course, that challenges VisiData's sheet-centric philosophy. Or maybe it should be a separate UI that's not sheet based, maybe something that floats on the screen and shows a sheet in the background as you highlight different items, like Alt-Tab in Windows/Linux.

So if a tree is not possible, then having the sheets ordered according to time of creation rather than last access would be good. Also, maybe including columns for times of creation and last open would allow us to customize what they're getting ordered by.
